### PR TITLE
[author] Add jquery-date-range-picker by package.json

### PR DESCRIPTION
**Pull request by holtkamp**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/longbill/jquery-date-range-picker
 * Official website (optional, not the repository):http://longbill.github.io/jquery-date-range-picker/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 493
   - Count of watchers: 36
   - Count of forks: 223
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [] Close corresponding issue in commit message
 * [] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Please fix the error first, the tool `./tools/fixFormat.js` may help you clean up the package.json

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help issuer review and fix this PR for further updates.

File: ajax/libs/jquery-date-range-picker/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "jquery": ">=1.3.2"
+  },
+  "optionalDependencies": {},
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/longbill/jquery-date-range-picker.git",
+    "basePath": "",
+    "files": [
+      "dist/daterangepicker.min.css",
+      "dist/jquery.daterangepicker.min.js"
```
Comment (x09326) [inline ajax/libs/jquery-date-range-picker/package.json]: Please modify the ```"basepath"``` to ```"dist"```.
Then modify the ```"files"``` to ```"**/*"```.

File: ajax/libs/jquery-date-range-picker/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "daterange",
+    "date",
+    "picker",
+    "range"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/longbill/jquery-date-range-picker.git"
+  },
+  "homepage": "https://github.com/longbill/jquery-date-range-picker",
```
Comment (x09326) [inline ajax/libs/jquery-date-range-picker/package.json]: We don't use github page as the homepage. 
If you don't have the homepage you can remove this field.

File: ajax/libs/jquery-date-range-picker/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery-date-range-picker",
+  "description": "jQuery Date Range Picker is a jQuery plugin that allows user to select a date range.",
+  "version": "0.5.1",
+  "license": "MIT",
+  "keywords": [
+    "jquery",
+    "date",
+    "daterange",
+    "date",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-date-range-picker/package.json]: `date` duplicate with line 8. Please remove it.

File: ajax/libs/jquery-date-range-picker/package.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "type": "git",
+    "url": "git://github.com/longbill/jquery-date-range-picker.git"
+  },
+  "homepage": "https://github.com/longbill/jquery-date-range-picker",
+  "author": "longbill",
+  "dependencies": {
+    "moment": ">=2.8.1",
+    "jquery": ">=1.3.2"
+  },
+  "optionalDependencies": {},
```
Comment (pvnr0082t) [inline ajax/libs/jquery-date-range-picker/package.json]: Please remove the empty field.

**Comment by holtkamp** [comment]
@cdnjs/intern2 I processed the comments, not sure I should/could smash the commits in one. The tests fail, but no idea why...

**Comment by PeterDaveHello** [comment]
@holtkamp I can squash commits for you, you just need to fix the error

**Comment by x09326** [comment]
@holtkamp 
I think because you didn't add ```"filename"``` field in your **package.json**.
So the test failed.

**Comment by holtkamp** [comment]
@x09326 , mmm, added the filename section, but tests keep failing reporting that the file does not exist...

```
jquery-date-range-picker 
      ✗ jquery-date-range-picker: filename from package.json exists 
        » ./ajax/libs/jquery-date-range-picker/0.5.1/dist/jquery.daterangepicker.min.js does not exist but is referenced in package.json! // valid-packages-test.js:104 
  ✗ Broken » 34472 honored ∙ 1 broken (8.914s) 
  npm ERR! Test failed.  See above for more details.

npm test returned exit code 1
```

Indeed, in this branch that file does not exist, but that should be 'generated' by the auto-update mechanism, right?

**Comment by PeterDaveHello** [comment]
If there is no real files, please remove "version", BTW, the filename should not contain "dist/"

**Comment by holtkamp** [comment]
Yeah, I added the ```dist``` part since I assumed that was why the file could not be found. Test is running again 🙏

**Comment by holtkamp** [comment]
@PeterDaveHello any idea why the autoupdate mechanism does not work?

CDNJS has up untill ```0.5.1``` while the source repository already arrived at ```0.6.0```...

https://cdnjs.com/libraries/jquery-date-range-picker
https://github.com/longbill/jquery-date-range-picker

**Comment by PeterDaveHello** [comment]
@holtkamp sorry that I broke the config when trying to import the files, going to fix it now.

**Pull request merged**